src/pyfmi/fmi3.pxd

```diff
@@ -24,17 +24,38 @@ cimport pyfmi.fmil_import as FMIL
 cimport pyfmi.fmil3_import as FMIL3
 cimport pyfmi.fmi_base as FMI_BASE
 
+cdef class ScalarVariable3:
```

Comment:
Suggestion: Rename to `ModelVariable`, or leave to separate effort.

Reply:
Good idea, fixed in latest commit

Reply:
Forgot to push a commit?

Reply:
Not sure what happened, but now I have pushed the change, see 9b5ed23